ensure_ocm_cli.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+ocmDownloads=$(curl -s https://api.github.com/repos/openshift-online/ocm-cli/releases/latest | grep "browser_download_url.*" | cut -d : -f 2,3 | tr -d \")
+osName=$(uname)
+osName="${osName,,}"
+hardware=$(uname -m)
+amd64="x86_64"
+if [[ "$hardware" = "$amd64" ]]; then
+    hardware="amd64"
+fi
```

Comment:
Maybe add a comment to make it clear which OS and hardwares are currently supported.